R/utils_render_rtf_redo.R

```diff
@@ -502,8 +502,7 @@ rtf_tbl_cell <- function(x,
 
 new_line <- function() {
 
-  rtf_text <-
-    list(rtf_text = rtf_key("line"))
+  rtf_text <- list(rtf_text = rtf_key("line"))
```

Comment:
Should not use a list

Reply:
Agree. And this function is so trivial I removed it (using `rtf_key("line")` in the one spot it's called).